### ISSUE TITLE
AttributeError: 'InMemorySessionManager' object has no attribute '_sessions'

### ISSUE THREAD

**Issue by Yagira** [eric-cc-su/PyAiml2#8]
```
>>> k = aiml.Kernel()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Yagira\AppData\Local\Programs\Python\Python35-32\lib\aiml\Kernel.py", line 42, in __init__
    self._sessionManager._addSession(self._globalSessionID)
  File "C:\Users\Yagira\AppData\Local\Programs\Python\Python35-32\lib\aiml\InMemorySessionManager.py", line 11, in _addSession
    if not sessionID in self._sessions:
AttributeError: 'InMemorySessionManager' object has no attribute '_sessions'
```

**Comment by lordkada** [comment]
@eric-cc-su: PR #9 should fix this (my bad... it was due to a bad code refactoring my side)

**Comment by eric-cc-su** [comment]
Resolved in PR #9 thanks @Yagira @lordkada
